core/src/main/java/org/pentaho/di/core/database/RemedyActionRequestSystemDatabaseMeta.java

```diff
@@ -33,7 +33,7 @@
 public class RemedyActionRequestSystemDatabaseMeta extends GenericDatabaseMeta implements DatabaseInterface {
   @Override
   public int[] getAccessTypeList() {
-    return new int[] { DatabaseMeta.TYPE_ACCESS_ODBC, DatabaseMeta.TYPE_ACCESS_JNDI };
+    return new int[] {DatabaseMeta.TYPE_ACCESS_JNDI };
```

Comment:
Considering the fact that "getDriverClass" and "getURL" methods return ODBC-only results; seems that Remedy Action Request System Database has to also be deprecated (and later dropped) similar to what is being done to "MS Access" and "dBase *".